packages/strapi-admin/admin/src/translations/en.json

```diff
@@ -229,11 +229,17 @@
   "Auth.form.register.confirmPassword.label": "Confirmation Password",
   "Auth.form.email.label": "Email",
   "Auth.form.email.placeholder": "johndoe@gmail.com",
+  "Auth.form.firstname.label": "First name",
+  "Auth.form.firstname.placeholder": "First name",
```

Comment:
You can use just one here for both label and placeholder as it is the same message `Auth.form.firstname: "First name"`